.projenrc.ts

```diff
@@ -74,12 +75,20 @@ const project = new awscdk.AwsCdkConstructLibrary({
   npmTokenSecret: 'NPM_TOKEN',
   npmAccess: NpmAccess.PUBLIC,
 
+  // Manual release, remove once NuGet Account is activated
+  releaseTrigger: ReleaseTrigger.manual(),
+
   publishToPypi: {
     distName: PUBLICATION_NAMESPACE+'.'+PROJECT_NAME,
     module: (PUBLICATION_NAMESPACE.replace(/-/g, '_'))+'.'+(PROJECT_NAME.replace(/-/g, '_')), // PEP 8, convert hypens
     // twineRegistryUrl: '${{ secrets.TWINE_REGISTRY_URL }}',
   },
 
+  publishToNuget: {
+    dotNetNamespace: PUBLICATION_NAMESPACE+'.'+PROJECT_NAME.replace(/-/g, '_'),
+    packageId: PUBLICATION_NAMESPACE+'.'+PROJECT_NAME.replace(/-/g, '_'),
```

Comment:
We may want to CamelCase this...let me test...probably add a function somewhere to define this CamelCasing

```suggestion
    dotNetNamespace: PUBLICATION_NAMESPACE.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {    if (+match === 0) return ""; return match.toUpperCase();  })+'.'+PROJECT_NAME.replace(/-/g, '_'),
    packageId: PUBLICATION_NAMESPACE.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {    if (+match === 0) return ""; return match.toUpperCase();  })+'.'+PROJECT_NAME.replace(/-/g, ' ').replace(/_/g, ' ').replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {    if (+match === 0) return ""; return match.toUpperCase();  }),
```

Reply:
Testing the package as-is first to see what it yields